src/app/application.cpp

```diff
@@ -512,6 +512,7 @@ void Application::initializeTranslation()
     Preferences* const pref = Preferences::instance();
     // Load translation
     QString localeStr = pref->getLocale();
+    QLocale::setDefault(QLocale(localeStr));
```

Comment:
>Probably, but if a user wants to change the language (different to their system), I would say that user should also accommodate to the new decimal separator.
when in Rome, do as the Romans do.

Thankfully, we use  QLocale::system().toString() and respect system locale settings for decimal mark.